src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
+  const [itemName, setItemName] = useState('');
+
+  /* Get items */
+  useEffect(() => {
+    const unsubscribe = onSnapshot(collection(db, 'Products'), (snapshot) => {
```

Comment:
According to your file `src\lib\api.js`, you have created a constant variable `colRef` that you might reuse or this variable has others purposes. 😄 [You might export this variable and import it in your component](https://velog.io/@_jouz_ryul/Exports-Imports-Modules-in-React).